### PR TITLE
fix(analytics): add missing usage events causing warning in logs

### PR DESCRIPTION
**Pull request by anshbansal**
Not having this was causing logs like this in gms which caused confusion.
```
INFO  c.l.m.k.t.DataHubUsageEventTransformer:74 - Invalid event type: HomePageViewEvent
WARN  c.l.m.k.DataHubUsageEventsProcessor:56 - Failed to apply usage events transform to record: ...
```

## Checklist

- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

**Comment by anshbansal** [comment]
Expected to fail with `ManuallyCreateLineageEvent` being missing

**Comment by anshbansal** [comment]
Code check did fail https://github.com/datahub-project/datahub/actions/runs/3989112410/jobs/6841138499 with 
```
Missing {'ManuallyCreateLineageEvent'} from DataHubUsageEventType.java. Please add
```

Added it now. So should pass now.

**Comment by anshbansal** [comment]
Code check passed now https://github.com/datahub-project/datahub/actions/runs/3989151394/jobs/6841229205
```
Passed
```

File: .github/scripts/check_event_type.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+with open("././datahub-web-react/src/app/analytics/event.ts") as ts_file:
+    for line in ts_file:
+        if '''Event,''' not in line:
+            continue
+        line = line.replace(",\n", "").replace(" ", "")
+        ts_events.add(line)
+
+ts_events_not_in_java = ts_events.difference(java_events)
+if len(ts_events_not_in_java) > 0:
+    print(f"Missing {ts_events_not_in_java} from DataHubUsageEventType.java. Please add")
```
Comment (jjoyce0510) [inline .github/scripts/check_event_type.py:21]: Thanks for adding this. We will need to watch these usage indexes as they continue to grow with all of these new events!

Overall, I like the idea of keeping these in sync.

**Comment by anshbansal** [comment]
Tested on local and don't see the warning showing up anymore in gms logs.

**Comment by anshbansal** [comment]
Work on fixing the failing cypress is being done in https://github.com/datahub-project/datahub/pull/7116

**Pull request merged**